arrow/src/main/java/org/apache/iceberg/arrow/vectorized/ArrowReader.java

```diff
@@ -136,6 +159,14 @@ public ArrowReader(TableScan scan, int batchSize, boolean reuseContainers) {
    * previous {@link ColumnarBatch} may be reused for the next {@link ColumnarBatch}.
    * This implies that the caller should either use the {@link ColumnarBatch} or deep copy the
    * {@link ColumnarBatch} before getting the next {@link ColumnarBatch}.
+   *
+   * <p>This method works for only when the following conditions are true:
```

Comment:
Nit: `<p>` normally goes on the line between paragraphs.

Reply:
fixed